system-addon/content-src/components/Search/Search.jsx

```diff
@@ -70,13 +77,13 @@ class Search extends React.Component {
         ref={this.onInputMount}
         title={this.props.intl.formatMessage({id: "search_web_placeholder"})}
         type="search" />
-        <button
-          id="searchSubmit"
-          className="search-button"
-          onClick={this.onClick}
-          title={this.props.intl.formatMessage({id: "search_button"})}>
-          <span className="sr-only"><FormattedMessage id="search_button" /></span>
-        </button>
+      <button
+        id="searchSubmit"
+        className="search-button"
+        onClick={this.onClick}
+        title={this.props.intl.formatMessage({id: "search_button"})}>
+        <span className="sr-only"><FormattedMessage id="search_button" /></span>
+      </button>
```

Comment:
Maybe leave out this change in case we just want to revert this commit down the line.